### PR TITLE
Eslint fixes

### PR DESCRIPTION
**Pull request by elohanlon**

### CODE REVIEW

File: app/javascript/src/hyacinth_v1/components/controlled_vocabularies/terms/TermForm.js [hunk 1]
```diff
@@ -123,11 +123,10 @@ function TermForm(props) {
           (() => {
             if (formType === 'edit') {
               return <PlainText value={uri} />;
-            } else if (termType === 'external' || termType === '') {
-              return <TextInput sm={inputColWidth} value={uri} onChange={v => setUri(v)} />;
-            } else {
-              return <ReadOnlyInput sm={inputColWidth} value={uri} />;
+            } if (termType === 'external' || termType === '') {
```
Comment (barmintor) [inline app/javascript/src/hyacinth_v1/components/controlled_vocabularies/terms/TermForm.js:126]: do you want a newline here?

**Review by barmintor — APPROVED** [review]
There's one change you might want (a newline in a suggestion here) but feel free to merge.

File: app/javascript/src/hyacinth_v1/components/controlled_vocabularies/terms/TermForm.js (hunk 1 above)
Comment (elohanlon) [reply app/javascript/src/hyacinth_v1/components/controlled_vocabularies/terms/TermForm.js:126]: Whoa, weird auto-reformat for my IDE to suggest/perform.  Yes I do!  Good catch, thanks!

[the remark this answers is not in the document]
Comment (elohanlon) [reply app/javascript/src/hyacinth_v1/components/controlled_vocabularies/terms/TermForm.js:126]: I also caught one other thing to fix in this file, so I'll make this change and another and force push.

**Pull request merged**